crates/libcgroups/src/v1/controller_type.rs

```diff
@@ -1,6 +1,6 @@
 use std::fmt::Display;
 
-#[derive(Hash, PartialEq, Eq, Debug, Clone)]
+#[derive(Hash, PartialEq, Eq, Debug, Clone, Copy)]
```

Comment:
Why we need `Copy`?

Reply:
This enum is essentially just a fancy number, so the performance of passing by reference and copying are the same

Reply:
Specifically, this allows us to do `*controller_type` instead of `controller_type.clone()` to turn a reference into an owned value